README.md

```diff
@@ -59,8 +59,7 @@ This will take care of downloading `cuCollections` from GitHub and making the he
 
 `cuCollections` depends on the following libraries:
 
-- [libcu++](https://github.com/NVIDIA/libcudacxx)
-- [CUB](https://github.com/thrust/cub)
+- [CUDA C++ Core Libraries (CCCL)](https://github.com/NVIDIA/cccl)
```

Comment:
we are not using `cccl` yet, right?

Reply:
The answer is complicated I guess. Technically speaking, all of the cccl facilities are now unified under this repo and the separate repos are deprecated. I know rapids-cmake is still in the process of migrating to a uniform version of all cccl libs, but that's a temporary detail. From a user's perspective this is more clear.

Reply:
okay. that's fair